canonical-web-and-design/maas-ui
# refactor(ui): Use Slider and Tabs components from react-components.

## Done

- Removed Slider and Tabs components from maas-ui in favour of the ones that exist in react-components 0.9.0

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Check that the tabs in the machine list and the KVM details pages still work.
- Check that they don't have a border underneath them.
- Check that you can still adjust CPU/memory overcommit using the sliders in the KVM configuration form
